lib/reporters/base.js

```diff
@@ -34,9 +34,9 @@ exports = module.exports = Base;
  * Enable coloring by default, except in the browser interface.
  */
 
-exports.useColors = process.env
-  ? (supportsColor || (process.env.MOCHA_COLORS !== undefined))
-  : false;
+exports.useColors = process.browser
+  ? false
+  : (supportsColor || (process.env.MOCHA_COLORS !== undefined));
```

Comment:
A ternary where one of the statements is a boolean constant always seems odd to me. How about

```javascript
!process.browser && (supportsColor || process.env.MOCHA_COLORS);
```